### ISSUE TITLE
Forbid single triple quotes

### ISSUE THREAD

**Issue by orsinium** [wemake-services/wemake-python-styleguide#1846]
# Rule request

## Thesis

```python
# bad
bad = '''
hello
'''

# good
good = """
hello
world
"""
```

## Reasoning

> For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in PEP 257.
> - [PEP-8](https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names)

I checked but I haven't found the rule in pycodestyle nor pylint.

**Comment by sobolevn** [comment]
It is covered by `flake8-quotes`.

```
ex.py

  1:1      WPS111 Found too short name: a < 2
  a = '''text'''
  ^

  1:5      Q001  Remove bad quotes from multiline string
  a = '''text'''
      ^

  1:5      WPS322 Found incorrect multi-line string
  a = '''text'''
```
